src/components/charts/costChart/costChart.tsx

```diff
@@ -59,16 +60,16 @@ interface State {
 
 class CostChart extends React.Component<CostChartProps, State> {
   private containerRef = React.createRef<HTMLDivElement>();
-  private resizeObserver: any = noop;
-  private navToggle: any = noop;
+  private observer: any = noop;
+
   public state: State = {
     hiddenSeries: new Set(),
     width: 0,
   };
 
   public componentDidMount() {
     this.initDatum();
-    this.initResizeObserve();
+    getResizeObserver(this.containerRef.current, this.handleResize);
```

Comment:
Need to save the observer, so we can release it later.

`this.observer = getResizeObserver(this.containerRef.current, this.handleResize);`